base/sort.jl

```diff
@@ -1560,7 +1599,7 @@ function sort!(v::FPSortable, a::Algorithm, o::DirectOrdering,
         t::Union{FPSortable, Nothing}=nothing)
     fpsort!(v, a, o, t)
 end
-function sort!(v::AbstractVector{T}, a::Algorithm, o::Perm{<:DirectOrdering,<:FPSortable}, 
+function sort!(v::AbstractVector{T}, a::Algorithm, o::Perm{<:DirectOrdering,<:FPSortable},
```

Comment:
`make` complains about the trailing whitespace here

Reply:
Yeah, that's my bad. I introduced that whitespace problem.